cosmos/cnho_stables.json

```diff
@@ -0,0 +1,57 @@
+{
+  "rpc": "https://rpc.cnho.io",
+  "rest": "https://api.cnho.io",
+  "nodeProvider": {
+    "name": "CNHO",
+    "email": "pr@cnho.io",
+    "website":"https://cnho.io/"
+  },
+  "chainId": "cnho_stables-1",
+  "chainName": "CNHO Stables",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cnho_stables/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "CNHO",
+    "coinMinimalDenom": "ucnho",
+    "coinDecimals": 6,
+ 
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cnho_stables/chain.png"
+  },
+  "bip44": {
+    "coinType": 60
+  },
+  "bech32Config": {
+    "bech32PrefixAccAddr": "cnho",
+    "bech32PrefixAccPub": "cnhopub",
+    "bech32PrefixValAddr": "cnhovaloper",
+    "bech32PrefixValPub": "cnhovaloperpub",
+    "bech32PrefixConsAddr": "cnhovalcons",
+    "bech32PrefixConsPub": "cnhovalconspub"
+  },
+  "currencies": [
+    {
+      "coinDenom": "CNHO",
+      "coinMinimalDenom": "ucnho",
+      "coinDecimals": 6,
+ 
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cnho_stables/chain.png"
+    }
+  ],
+  "feeCurrencies": [
+    {
+      "coinDenom": "CNHO",
+      "coinMinimalDenom": "ucnho",
+      "coinDecimals": 6,
+ 
+      "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/cnho_stables/chain.png",
+      "gasPriceStep": {
+        "low": 0.005,
+        "average": 0.01,
+        "high": 0.02
+      }
+    }
+  ],
+  "features": [    "cosmwasm",
+  "eth-address-gen",
+  "eth-key-sign"],
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/cnhostables"
```

Comment:
Could you remove this line, since the page is not supported dynamically?

Reply:
Ok i remove it and try